api-docs/v2/general-api-info/limits.rst

```diff
@@ -13,7 +13,8 @@ are fixed.
 Rate limits
 ~~~~~~~~~~~
 
-Rate limits are shown below for API operations by resource. They are displayed in a simplified, wild-card style format.
+Rate limits are shown below for API operations by resource. They are displayed in a 
+simplified, wild-card style format.
```

Comment:
I think `simplified format` would be sufficient here. They're not _really_ wildcards, but I think it makes sense the way they are without any additional explanation.